src/views/Inventory.vue

```diff
@@ -0,0 +1,161 @@
+            <ion-toggle disabled color="secondary" slot="end" checked />
+          </ion-item>
+          <ion-item lines="none">
+            <ion-label class="ion-text-wrap">
+              <p>{{ $t("Realtime adjustments allow HotWax Commerce to push new inventory to Shopify in realtime. These events include receiving , cycle counts, variances, and return.") }}</p>
+            </ion-label>
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ $t("Hard sync") }}</ion-label>
+            <ion-select :interface-options="customPopoverOptions" interface="popover">
+              <ion-select-option>Hourly</ion-select-option>
+              <ion-select-option>Every 6 hours</ion-select-option>
+              <ion-select-option>Nightly</ion-select-option>
+              <ion-select-option>Daily</ion-select-option>
+            </ion-select>
+          </ion-item>
+          <ion-item lines="none">
+            <ion-label class="ion-text-wrap">
+              <p>{{ $t("Performing a hard sync from Shopify to HotWax Commerce is useful for eliminating any discrepencies.") }}</p>
+            </ion-label>
+          </ion-item>
+           <ion-item>
+            <ion-label>{{ $t("Realtime POS sales") }}</ion-label>
+            <ion-toggle disabled color="secondary" slot="end" checked />
+          </ion-item>
+           <ion-item>
+            <ion-label>{{ $t("Reserve for completed orders") }}</ion-label>
+            <ion-toggle disabled color="secondary" slot="end" checked />
+          </ion-item>
+          <ion-item lines="none">
+            <ion-label class="ion-text-wrap">
+              <p>{{ $t("Reserving inventory for completed orders allows POS sales to accurately deduct inventory in HotWax Commerce.") }}</p>
+            </ion-label>
+          </ion-item>
+          <ion-item lines="none">
+            <ion-label class="ion-text-wrap">
+              <p>{{ $t("When importing historical completed orders, reservation should be turned off.") }}</p>
+            </ion-label>
+          </ion-item>
+        </ion-card>
+      </main>
+    </ion-content>
+  </ion-page>
+</template>
+
+<script lang="ts">
+import {
+  IonCard,
+  IonContent,
+  IonHeader,
+  IonItem,
+  IonLabel,
+  IonPage,
+  IonRadioGroup,
+  IonRadio,
+  IonSelect,
+  IonSelectOption,
+  IonTitle,
+  IonToggle,
+  IonToolbar,
+} from '@ionic/vue';
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'Inventory',
+  components: {
+    IonCard,
+    IonContent,
+    IonHeader,
+    IonItem,
+    IonLabel,
+    IonPage,
+    IonRadioGroup,
+    IonRadio,
+    IonSelect,
+    IonSelectOption,
+    IonTitle,
+    IonToggle,
+    IonToolbar,
+  },
+  setup() {
+    const customPopoverOptions: any = {
+    header: 'Title',
+    showBackdrop: false
+    }
+    return {
+      customPopoverOptions  
+    }  
+  }
+});
+</script>
+
+<style scoped>
+main {
+  display: flex;  
+  align-items: flex-start;
+  max-width: 742px;
```

Comment:
I don't think you need to define width here. There is a way to do this without defining explicit width.

Reply:
Done